src/scenes/Root.tsx

```diff
@@ -18,6 +20,33 @@ const useStyles = makeStyles(() => ({
 
 export const Root = () => {
   useStyles();
+
+  const navigate = useNavigate();
+  const [bootstrapping, setBootstrapping] = useState(true);
+  const { setUserSession } = useUserSessionContext();
+  const { data, loading: loadingSession, error } = useUserSessionQuery();
+
+  React.useEffect(() => {
+    // If the session query has finished loading
+    if (!loadingSession || error) {
+      // Check if an existing session exists and add it to session context
+      if (data?.session.user) {
+        setUserSession(data.session.user);
+      } else {
+        // Otherwise go to the login scene
+        navigate('/login');
```

Comment:
We don't want to do this here. There are anonymous routes, like forgot/reset password.

A `useRequireLoggedIn` hook might not be a bad abstraction though.

Reply:
what should we do here in the meantime? just allow any route to load until we build the `useRequireLoggedIn` hook?

Reply:
Just added a "TODO" for now, let me know how to proceed!

Reply:
Yeah we don't have to enforce it yet.

Reply:
👍